client/snapper.cc

```diff
@@ -1665,6 +1669,17 @@ main(int argc, char** argv)
     if ((opt = opts.find("no-dbus")) != opts.end())
 	no_dbus = true;
 
+    if ((opt = opts.find("root")) != opts.end())
+    {
+	target_root = opt->second;
+        if (!no_dbus)
```

Comment:
yes, because it use find, so it search all opts in previous step, so test passed:
```
pepan@linux-5mqx:/data/prace/snapper/client> ./snapper --root=/tmp --no-dbus list-configs
Listing configs failed (sysconfig-file not found).
pepan@linux-5mqx:/data/prace/snapper/client> ./snapper --no-dbus list-configs
Config | Subvolume
-------+----------

```